src/amo/components/Category/index.js

```diff
@@ -58,7 +59,9 @@ export class CategoryBase extends React.Component {
       page: parsePage(location.query.page),
     };
     const pathname = `/${params.visibleAddonType}/${categorySlug}/`;
-    const paginationQueryParams = { page: filters.page };
+    const paginationQueryParams = convertFiltersToQueryParams({
```

Comment:
This doesn't change anything right now because `page` is one of the few `filters` that has the same key as in `queryParams` but this is what we should be doing, so I did it here.